src/TypeSpec.Extension/Emitter.Csharp/src/lib/operation.ts

```diff
@@ -116,19 +112,28 @@ export function loadOperation(
             typespecParameters.body.type
         );
         if (effectiveBodyType.kind === "Model") {
-            if (effectiveBodyType.name !== "") {
-                parameters.push(
-                    loadBodyParameter(sdkContext, effectiveBodyType)
-                );
-            } else {
-                effectiveBodyType.name = `${capitalize(op.name)}Request`;
-                let bodyParameter = loadBodyParameter(
-                    sdkContext,
-                    effectiveBodyType
-                );
+            let bodyParameter = loadBodyParameter(
+                sdkContext,
+                effectiveBodyType
+            );
+            if (effectiveBodyType.name == "") {
                 bodyParameter.Kind = InputOperationParameterKind.Spread;
-                parameters.push(bodyParameter);
             }
+            // TODO: remove this after https://github.com/Azure/typespec-azure/issues/69 is resolved
+            // workaround for alias model
+            if (bodyParameter.Type.Name == "") {
```

Comment:
The `bodyParameter.Type.Name` should not be empty because when load the body type via calling ` function getInputModelForModel(m: Model): InputModelType 
` it will use the `generatedName` for anonymous model, so the model name should not be empty. 
is there any bug in `getSdkModel` or anonymous model name handling?

Reply:
It's an known issue in TCGC that it won't handle alias model: https://github.com/Azure/typespec-azure/issues/69